packages/tests/package.json

```diff
@@ -43,19 +41,7 @@
       "clean": "if-file-deleted"
     },
     "test": {
-      "dependencies": [
-        "test:dev",
-        "test:prod"
-      ]
-    },
-    "test:dev": {
-      "command": "node ./run-web-tests.js",
-      "dependencies": [
-        "build"
-      ]
-    },
-    "test:prod": {
-      "command": "MODE=prod node ./run-web-tests.js",
+      "command": "node ./run-web-tests.js && MODE=prod node ./run-web-tests.js",
```

Comment:
We could also set `WIREIT_PARALLEL=1` after the build step, but before the test step.

Reply:
Merging w/o this for now. It would mean we could get sauce tunnel errors if we ran this locally. I think we'll be refactoring this still...